packages/kn-plugin-workflow/pkg/command/deploy_undeploy_common.go

```diff
@@ -122,14 +122,14 @@ func generateManifests(cfg *DeployUndeployCmdConfig) error {
 	supportFileExtensions := []string{metadata.JSONExtension, metadata.YAMLExtension, metadata.YMLExtension}
 
 	fmt.Println("🔍 Looking for specs files...")
-	files, err = common.FindFilesWithExtensions(cfg.SpecsDir, supportFileExtensions)
+	minifiedfiles, err := common.NewMinifier(&common.OpenApiMinifierParams{
```

Comment:
Then if you accept my other suggestion we can have something like this:

```suggestion
	minifiedfiles, err := specs.NewMinifier(&common.OpenApiMinifierParams{
```